# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/base/strings/safe_sprintf_unittest.cc
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/base/strings/safe_sprintf_unittest.cc
--- a/security/sandbox/chromium/base/strings/safe_sprintf_unittest.cc	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/base/strings/safe_sprintf_unittest.cc	Wed Jun 12 11:10:48 2019 +0100
@@ -384,14 +384,16 @@
   size_t out_sz = sz;
   size_t len;
   for (std::unique_ptr<char[]> perfect_buf;;) {
-    size_t needed = SafeSNPrintf(out, out_sz,
+    size_t needed =
+        SafeSNPrintf(out, out_sz,
 #if defined(NDEBUG)
-                            "A%2cong %s: %d %010X %d %p%7s", 'l', "string", "",
+                     "A%2cong %s: %d %010X %d %p%7s", 'l', "string", "",
 #else
-                            "A%2cong %s: %%d %010X %d %p%7s", 'l', "string",
+                     "A%2cong %s: %%d %010X %d %p%7s", 'l', "string",
 #endif
-                            0xDEADBEEF, std::numeric_limits<intptr_t>::min(),
-                            PrintLongString, static_cast<char*>(NULL)) + 1;
+                     0xDEADBEEF, std::numeric_limits<intptr_t>::min(),
+                     PrintLongString, static_cast<char*>(nullptr)) +
+        1;
 
     // Various sanity checks:
     // The numbered of characters needed to print the full string should always